jme3-core/src/main/java/com/jme3/system/AppSettings.java

```diff
@@ -1507,4 +1508,36 @@ public int getDisplay() {
     public void setDisplay(int mon) {
         putInteger("Display", mon);
     }
+
+    /**
+     * Sets the native platform to be used to create the GL context.
+     * 
+     * <p>
+     * This only affects Linux distributions or derivatives that use a Wayland session in conjunction 
+     * with X11 via the XWayland bridge, which enables or disables GLX for window positioning and/or 
+     * icon configuration.
+     * </p>
+     *
+     * <p>
+     * <strong>NOTE:</strong> Note that disabling this option uses GLX (native X11) instead of EGL (native WL).
+     * </p>
+     * 
+     * @param nplaf true if you want to enable GLX, otherwise false when using EGL (native)
+     */
+    public void setX11PlatformPreferred(boolean nplaf) {
+        put("X11PlatformPreferred", nplaf);
+    }
+    
+    /**
+     * Gets what type of platform is being used.
+     * 
+     * <p>
+     * Only valid on Linux distributions or derivatives that support Wayland, where it indicates whether GLX or EGL is enabled.
+     * </p>
+     * 
+     * @return returns true if GLX is enabled, otherwise false if used in EGL (native)
+     */
+    public boolean isX11PlatformPreferred() {
+        return getBoolean("X11PlatformPreferred");
```

Comment:
Please update javadoc as follow:

    /**
     * Determines which native platform is preferred for GL context creation on Linux distributions.
     * <p>
     * This setting is only valid on Linux distributions or derivatives that support Wayland,
     * and it indicates whether GLX (native X11) or EGL (native Wayland) is enabled for the GL context.
     * </p>
     *
     * @return True if GLX is preferred, otherwise false if EGL is preferred (native Wayland).
     */